mintpy/utils/isce_utils.py

```diff
@@ -333,6 +346,159 @@ def extract_stripmap_metadata(meta_file):
     return meta, frame
 
 
+def extract_alosStack_metadata(meta_file, geom_dir):
+
+    import isce
+    import isceobj
+    from isceobj.Planet.Planet import Planet
+
+    track = load_track(os.path.dirname(meta_file), os.path.basename(meta_file).strip('.track.xml'))
+    rlooks, alooks, width, length = extract_image_size_alosStack(geom_dir)
+    spotlightModes, stripmapModes, scansarNominalModes, scansarWideModes, scansarModes = alos2_acquisition_modes()
+
+    meta = {}
+    meta['prf']             = track.prf
+    meta['startUTC']        = track.sensingStart + datetime.timedelta(seconds=(alooks-1.0)/2.0*track.azimuthLineInterval)
+    meta['stopUTC']         = meta['startUTC'] + datetime.timedelta(seconds=(length-1)*alooks*track.azimuthLineInterval)
+    meta['radarWavelength'] = track.radarWavelength
+    meta['startingRange']   = track.startingRange + (rlooks-1.0)/2.0*track.rangePixelSize
+    meta['passDirection']   = track.passDirection.upper()
+    meta['polarization']    = track.frames[0].swaths[0].polarization
+    #meta['trackNumber']     = track.trackNumber
+    #meta['orbitNumber']     = track.orbitNumber
+
+    meta['PLATFORM'] = sensor.standardize_sensor_name('alos2')
+
+    sensingMid = meta['startUTC'] + datetime.timedelta(seconds=(meta['stopUTC']-meta['startUTC']).total_seconds()/2.0)
+    time_seconds = (sensingMid.hour * 3600.0 +
+                    sensingMid.minute * 60.0 +
+                    sensingMid.second)
+    meta['CENTER_LINE_UTC'] = time_seconds
+
+    peg = track.orbit.interpolateOrbit(sensingMid, method='hermite')
+    Vs = np.linalg.norm(peg.getVelocity())
+    meta['azimuthPixelSize'] = Vs * track.azimuthLineInterval
+    meta['rangePixelSize'] = track.rangePixelSize
+
+    azb = track.prf * 0.8
```

Comment:
Could you write the full name for this variable `azb` in the comments for better understanding?

Reply:
that is azimuth bandwidth

Reply:
I see, thanks.